add-on/src/recovery/recovery.html

```diff
@@ -0,0 +1,34 @@
+        </div>
+        <h1 id="header-text" class="pt2 pt0-l ma0 f3 fw2 montserrat aqua ttu flex-none no-select"></h1>
+      </header>
+      <main class="bg-white pv3 pa3 pa4-l flex-grow-1">
+          <div id="root"></div>
+      </main>
+      <footer class="flex pa3 justify-end">
+        <a
+          class="f5 fw4 no-underline aqua"
+          id="learn-more"
+          href="https://docs.ipfs.tech/how-to/companion-node-types/#external"
```

Comment:
More useful if we link to the top:

```suggestion
          href="https://docs.ipfs.tech/how-to/companion-node-types/"
```